packages/terra-avatar/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "terra-avatar",
+  "main": "lib/Avatar.js",
+  "version": "0.0.0",
+  "description": "The terra-avatar component displays an avatar, which can be either an image or text, in a circular frame. If neither an image or text is provided, a variant-based fallback icon is used.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-avatar",
+    "Avatar",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
```

Comment:
remove terra-base and terra-mixins from the peer dependencies.

Reply:
Can you explain why these need to be removed? Looking quickly at the package.json's for [terra-arrange](https://github.com/cerner/terra-core/blob/master/packages/terra-arrange/package.json#L37), [terra-form-textarea](https://github.com/cerner/terra-core/blob/master/packages/terra-form-textarea/package.json#L29), and [terra-button](https://github.com/cerner/terra-core/blob/master/packages/terra-button/package.json#L37) all seem to show terra-base as a dependency?

Reply:
Let's keep this as is, it aligns with how we handle this in other terra packages that @dylan-sftwr linked to.